src/Fulma/Components/Navbar.fs

```diff
@@ -19,6 +19,9 @@ module Navbar =
         /// Add `is-fixed-bottom` class
         /// You also need to add `has-navbar-fixed-bottom` to your html tag
         | [<CompiledName("is-fixed-bottom")>] IsFixedBottom
+        /// Add `is-spaced` class
+        /// Sets Top and Bottom paddings with 1 rem, Left and Right paddings with 2 rem
```

Comment:
I don't think we can add space info in the documentation.

The `1rem` and `2rem` can be changed using variables.

Reply:
"Increases horizontal and vertical padding of the navbar" would be a better fit then.

Reply:
done